app/assets/stylesheets/_cookie-banner.scss

```diff
@@ -0,0 +1,56 @@
+.cookies-banner {
+  background-color: $lightgrey-10;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;  
+  border-radius: 4px;
+  margin: 0 auto;
+  padding: 10px;
+  left: 20px;
+  right: 20px;
+  max-width: fit-content;
+  bottom: -400px;
+  box-shadow: $lightgrey-16 0px 0px 12px 0px;
+  z-index: 1000;
+  visibility: hidden;
+  transition: all 0.8s ease-in-out;  
+
+  .cookies-banner__msg {
+    margin: 0 20px 0 0;
+    text-align: justify;
```

Comment:
I don't have a strong opinion on that, but have you tried with `text-align: left`? I think with justified text it may look odd on some screens like this first line here. What do you think? Have you found it better?
![image](https://user-images.githubusercontent.com/1292945/172849374-449956f5-33d1-495e-8053-38c44285d361.png)